requirements_dev.txt

```diff
@@ -8,7 +8,7 @@ tabulate
 numpy>=1.9.1
 scipy>=0.9.0
 blessings>=1.5.1
-pandas>=0.15.2
+pandas==0.18.1
```

Comment:
ok. But let's merge this in when its ready, then #168. We can update the version in #168.